protos/telemetry_server/telemetry_server.proto

```diff
@@ -294,6 +314,39 @@ message RcStatus {
     float signal_strength_percent = 3 [(mavsdk.options.default_value)="NaN"]; // Signal strength (range: 0 to 100, NaN if unknown)
 }
 
+// Cellular modem status type.
+message CellularStatus {
+    uint32 status = 1;
```

Comment:
By the way, this is how I added status flags for the winch as a separate proto message: https://github.com/mavlink/MAVSDK-Proto/pull/308/files#diff-c29712674081664a7c33c32b2a4772848836bbbe76c1f1de17d5cb82b9f512acR73-R88

I.e. all bit flags from MAVLink became Booleans in Proto.